git_aggregator/config.py

```diff
@@ -27,10 +27,27 @@ def get_repos(config, force=False):
             directory = os.path.abspath(directory)
         repo_dict = {
             'cwd': directory,
-            'defaults': repo_data.get('defaults', dict()),
             'force': force,
+            'defaults': dict(),
```

Comment:
does this change the behaviour for `defaults`?

Reply:
No, it doesn't change anything. It's just there to set a default value for the `repo_dict`